packages/app-mobile/components/screens/ConfigScreen/ConfigScreen.tsx

```diff
@@ -351,28 +449,38 @@ class ConfigScreenComponent extends BaseScreenComponent<ConfigScreenProps, Confi
 						)
 					}
 				</View>,
+				[label, description],
 			);
 		}
 
 		if (section.name === 'tools') {
-			settingComps.push(this.renderButton('profiles_buttons', _('Manage profiles'), this.manageProfilesButtonPress_));
-			settingComps.push(this.renderButton('status_button', _('Sync Status'), this.syncStatusButtonPress_));
-			settingComps.push(this.renderButton('log_button', _('Log'), this.logButtonPress_));
-			settingComps.push(this.renderButton('fix_search_engine_index', this.state.fixingSearchIndex ? _('Fixing search index...') : _('Fix search index'), this.fixSearchEngineIndexButtonPress_, { disabled: this.state.fixingSearchIndex, description: _('Use this to rebuild the search index if there is a problem with search. It may take a long time depending on the number of notes.') }));
+			addSettingButton('profiles_buttons', _('Manage profiles'), this.manageProfilesButtonPress_);
+			addSettingButton('status_button', _('Sync Status'), this.syncStatusButtonPress_);
+			addSettingButton('log_button', _('Log'), this.logButtonPress_);
+			addSettingButton('fix_search_engine_index', this.state.fixingSearchIndex ? _('Fixing search index...') : _('Fix search index'), this.fixSearchEngineIndexButtonPress_, { disabled: this.state.fixingSearchIndex, description: _('Use this to rebuild the search index if there is a problem with search. It may take a long time depending on the number of notes.') });
 		}
 
 		if (section.name === 'export') {
-			settingComps.push(<NoteExportButton key='export_as_jex_button' styles={this.styles()} />);
-			settingComps.push(<ExportDebugReportButton key='export_report_button' styles={this.styles()}/>);
-			settingComps.push(<ExportProfileButton key='export_data' styles={this.styles()}/>);
+			addSettingComponent(
+				<NoteExportButton key='export_as_jex_button' styles={this.styles()} />,
+				[exportButtonTitle(), exportButtonDescription()],
+			);
+			addSettingComponent(
+				<ExportDebugReportButton key='export_report_button' styles={this.styles()}/>,
+				exportDebugReportTitle(),
+			);
+			addSettingComponent(
+				<ExportProfileButton key='export_data' styles={this.styles()}/>,
+				exportProfileButtonTitle(),
+			);
```

Comment:
I don't like how this is structured. Currently, `NoteExportButton`, `ExportDebugReportButton`, and `ExportProfileButton` all export their components **and descriptions**. It might make more sense to export a function that returns an object with the following format:
```
{
    searchText: string;
    component: ReactElement;
}|null
```